www/activities/diary/js/diary.js

```diff
@@ -170,6 +170,31 @@ app.Diary = {
     el.innerText = dateString;
   },
 
+  sendStatistics: async function() {
+    let entry = await this.getEntryFromDB(); // Get diary entry from DB
+    let totalNutrition;
+
+    // Populate groups and get overal nutrition
+    if (entry) {
+      await this.populateGroups(entry);
```

Comment:
Why do the groups need to be populated here?

Reply:
done